src/shared/utils/util.ts

```diff
@@ -494,6 +494,10 @@ export class Util {
     );
   }
 
+  static blankCentre(value: string, visibleLength = 4): string {
+    return value.slice(0, visibleLength) + this.blankStart(value, visibleLength);
+  }
```

Comment:
`Center`?

Reply:
Both possible, the one is british english and the other one american 😁

Reply:
True, we normally use American english 😉